fbw-common/src/systems/navdata/client/backends/Msfs/Mapping.ts

```diff
@@ -1449,7 +1449,12 @@ export class MsfsMapping {
       // eslint-disable-next-line prefer-destructuring
       let nextIcao: string | undefined = route.nextIcao;
 
+      let iterationCount = 0;
+
       while (previousIcao?.trim() || nextIcao?.trim()) {
+        if (++iterationCount > 100) {
```

Comment:
I did a quick check; we should make this 250 to be on the safe side for all cases.